paths.json

```diff
@@ -22,6 +22,12 @@
     "/content/world-bank/corporate/ext/fr/data/placeholders:/ext/fr/placeholders.json",
     "/content/world-bank/corporate/ext/es/data/placeholders:/ext/es/placeholders.json",
     "/content/world-bank/corporate/ext/ar/data/placeholders:/ext/ar/placeholders.json",
+    "/content/world-bank/corporate/ext/en/data/metadata:/ext/en/metadata.json",
```

Comment:
@singhpritam18 I believe metadata is Global configuration which is already added. What usecase you are trying with metadata?

Reply:
Thiis to to add JSON json-ld schema for each url Page as described here https://main--wilson--wilson-ecommerce.aem.page/en-us/custom-config/product-metadata.json